.changeset/shy-ears-report.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sveltejs/kit': minor
+---
+
+feat: Support caching of responses with `Vary` header (except for `Vary: *`)
+
+fix: Include `Vary: Accept` header to fix browser caching of adjacent pages and endpoints
```

Comment:
```suggestion
feat: support caching of responses with `Vary` header (except for `Vary: *`)

fix: include `Vary: Accept` header to fix browser caching of adjacent pages and endpoints
```

idk why but we don't capitalize these  <picture data-single-emoji=":lol:" title=":lol:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/lol/5986e156eca2154f.png" alt=":lol:" align="absmiddle"></picture>